Oneirocom/Magick
# Jakobs/agent deploy UI

## What Changed:

### Live Agents! 
* We now have the concept of two agents. A draft and a published. A user currently cannot edit a live running agent instead, they should edit their draft agent spells and then proceed to click the publish to live agent.

* Three conditions for loading exist. 
1. a user has a draft and a live agent and thus the app loads up normally and sets user to their draft agent. 
2. an existing user who has only a live agent, they get a draft agent made from their live agent, a release is made for the live agent so as to set the user in the two agent paradigm.
3. no agents exist and thus both draft and live agent is created for the user

* We have a spell release selector in the config page
* spell drawer now only shows spells relative to the release your agent is pinned too.
* editing of live agent is blocked
* modals informing creation of agent
* when a release happens the root spell of the new pinned release is set to the agent so as to load up the propper spells in the runner
* add spellReleases table
* unified some time stamp styles
* no longer can create more agents than the two that you have

## How to test:
* try loading existing projects
* ensure that any existing and working live agents are actually still working
How can the reviewer test your changes and validate that they work as described?
